src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/Performance/CSharpUseSpanClearInsteadOfFIll.Fixer.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Composition;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.NetCore.Analyzers.Performance;
+
+namespace Microsoft.NetCore.CSharp.Analyzers.Performance
+{
+    /// <summary>
+    /// CA1849: C# implementation of use Span.Clear instead of Span.Fill(default)
+    /// Implements the <see cref="CodeFixProvider" />
+    /// </summary>
+    /// <seealso cref="UseSpanClearInsteadOfFillFixer"/>
+    [ExportCodeFixProvider(LanguageNames.CSharp), Shared]
+    public class CSharpUseSpanClearInsteadOfFillFixer : UseSpanClearInsteadOfFillFixer
```

Comment:
VB fixer and tests are missing.

Reply:
VB can't use `Span`. It's actually not applicable to VB.

Reply:
@huoyaoyuan The analyzer is marked as supporting VB. That should be fixed then if it's only applicable to C#.

Reply:
I'm doing the same thing with CA1831. It's providing a analyzer for both language, and only a fixer for C#.

Reply:
@huoyaoyuan Can a diagnostic be reported in VB? If yes, then tests are needed. If no, then I don't see why providing the analyzer for VB is needed.

Reply:
It should be able to report diagnostic. The test framework isn't capable to test analyzer with fixer.
Anyway, I'm aligning with UseAsSpanInsteadOfRangeIndexer.

Reply:
> The test framework isn't capable to test analyzer with fixer.

Assuming you meant to say "without fixer", then no, the test framework can test analyzers without fixers. You only need to use `Microsoft.CodeAnalysis.Testing.EmptyCodeFixProvider`.

Reply:
> I'd want to remove the declaration of VB support, since there's always error BC30668. Is it OK?

I think it is reasonable to move VB support and tests to a separate PR. However, if we want to completely drop VB support for this analyzer, a confirmation is needed from the team on https://github.com/dotnet/runtime/issues/33813.
@buyaa-n @carlossanlop what do you think?

Reply:
I think it makes sense to drop VB support. As noted, VB doesn't have anything beyond bare minimum span support.